python/ingestion/primary_care_access_to_gcs.py

```diff
@@ -1,18 +1,20 @@
 from .di_url_file_to_gcs import download_first_url_to_gcs
-from .constants import _STATE_NAMES
-import logging
-import os
-from google.cloud import storage
-import google.cloud.exceptions
-import requests
+from .constants import STATE_NAMES
 
 
 _FILEPATH = '{}-{}.xlsx'
-_URL1 = 'https://www.countyhealthrankings.org/sites/default/files/media/document/2020 County Health Rankings {} Data - v1_0.xlsx'
-_URL2 = 'https://www.countyhealthrankings.org/sites/default/files/media/document/2020 County Health Rankings {} Data - v1.xlsx'
+_URL1 = """https://www.countyhealthrankings.org/sites/default/files/media/document/
+    2020 County Health Rankings {} Data - v1.xlsx"""
+_URL2 = """https://www.countyhealthrankings.org/sites/default/files/media/document/
+    2020 County Health Rankings {} Data - v1_0.xlsx"""
```

Comment:
I don't think this works - it adds extra spacing into the string because of the indent. See https://stackoverflow.com/questions/10660435/pythonic-way-to-create-a-long-multi-line-string

I think you want to do:
```
_URL1 = ("first part of string"
         "second part of string")
```